irf/gadf/time.py

```diff
@@ -0,0 +1,127 @@
+from pandas.api.types import is_datetime64_any_dtype
+import pandas as pd
+from astropy.time import Time
+import astropy.units as u
+import numpy as np
+import fact
+
+# the timestamps in the fits files need a reference MJD time according to some (ogip?) standard.
+# The exact value is not important for anything.
+MJDREF = 55835  # MJD sometime near FACT's first light. 2011-10-01T00:00:00 UTC
+
+
+
+# These values are constants like the location of the telescope and its altitude ASL.
+# As well as the TIMESYS keyword which is always fixed to 'utc' for FACT.
+TIME_INFO = {
+    'MJDREFI': MJDREF,
+    'MJDREFF': 0,
+    'TIMEREF': 'local',
+    'TIMESYS': 'utc',
+    'GEOLON': -17.890701,
+    'GEOLAT': 28.761795,
+    'ALTITUDE': 2199.4,
+    'TIMEUNIT': 's',
+}
+
+
+def timestamp_to_mjdref(timestamp, mjdref=MJDREF, location=fact.instrument.constants.LOCATION):
```

Comment:
I think astropy can now handle timestamps for fits serialization: http://docs.astropy.org/en/stable/io/fits/usage/table.html#fits-table-with-time-columns